File: app/src/main/java/igrek/forceawaken/logger/LoggerFactory.java
package igrek.forceawaken.logger;

public class LoggerFactory {
	
	public static final LogLevel CONSOLE_LEVEL = LogLevel.TRACE;
	public static final String LOG_TAG = "dupa";
	public static final boolean SHOW_EXCEPTIONS_TRACE = true;
	public static final LogLevel SHOW_TRACE_DETAILS_LEVEL = LogLevel.DEBUG;
	
	public static Logger getLogger() {
		return new Logger();
	}
	
}

File: app/src/main/java/igrek/forceawaken/info/logger/LoggerFactory.kt
package igrek.forceawaken.info.logger

object LoggerFactory {

    val CONSOLE_LEVEL = LogLevel.TRACE

    val SHOW_TRACE_DETAILS_LEVEL = LogLevel.FATAL

    const val LOG_TAG = "dupa"

    val logger: Logger
        get() = Logger()

}
